core/imageroot/usr/local/agent/bin/module-backup

```diff
@@ -101,11 +121,14 @@ if os.path.isfile(install_dir + "/etc/state-exclude.conf"):
 if agent.run_restic(rdb, repository, repopath, [], ["snapshots"], stdout=subprocess.DEVNULL).returncode == 0:
     print(f"Repository {repository} is present at path {repopath}", file=sys.stderr)
 else:
-    print(f"Initializing repository {repository} at path {repopath}", file=sys.stderr)
-    agent.run_restic(rdb, repository, repopath, [], ["init"]).check_returncode()
+    try:
+        print(f"Initializing repository {repository} at path {repopath}", file=sys.stderr)
+        agent.run_restic(rdb, repository, repopath, [], ["init"]).check_returncode()
+    except subprocess.CalledProcessError as ex:
+        print("[ERROR] restic init failed.", ex, file=sys.stderr)
+        errors += 1
```

Comment:
Replace with:
    backup['errors'] += 1

Reply:
I would prefer to stick to errors if you could accept